src/models/event.ts

```diff
@@ -1070,33 +1148,33 @@ utils.extend(MatrixEvent.prototype, {
      *
      * @param {string} eventId the new event id
      */
-    updateAssociatedId(eventId) {
+    public updateAssociatedId(eventId: string): void {
         const relation = this.getRelation();
         if (relation) {
             relation.event_id = eventId;
         } else if (this.isRedaction()) {
             this.event.redacts = eventId;
         }
-    },
+    }
 
     /**
      * Flags an event as cancelled due to future conditions. For example, a verification
      * request event in the same sync transaction may be flagged as cancelled to warn
      * listeners that a cancellation event is coming down the same pipe shortly.
      * @param {boolean} cancelled Whether the event is to be cancelled or not.
      */
-    flagCancelled(cancelled = true) {
+    public flagCancelled(cancelled = true): void {
         this._isCancelled = cancelled;
-    },
+    }
 
     /**
      * Gets whether or not the event is flagged as cancelled. See flagCancelled() for
      * more information.
      * @returns {boolean} True if the event is cancelled, false otherwise.
      */
-    isCancelled() {
+    isCancelled(): boolean {
```

Comment:
This can be removed and `_isCancelled` made public

Reply:
that changes the signature from a function to a field which would be a breaking change